ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -84,31 +83,9 @@ pub async fn test_populated_offer(peertest_config: PeertestConfig, peertest: &Pe
     validate_portal_offer(result, peertest);
 
     // Check if the stored content item in bootnode's DB matches the offered
-    let received_content_value = wait_for_content(ipc_client, content_key).await;
+    let received_content_value = ipc_client.local_content(content_key).await.unwrap();
     assert_eq!(
         content_value, received_content_value,
         "The received content {received_content_value:?}, must match the expected {content_value:?}",
     );
 }
-
-/// Wait for the content to be transferred
-async fn wait_for_content(
```

Comment:
Not sure about the status of this function. Is it being used? I was not clear on how to retain the function in the context of removing the `::Unknown` variant